packages/carousel/src/Carousel.stories.js

```diff
@@ -44,13 +48,30 @@ export default {
 const BasicTemplate = (args) => (
   <Carousel {...args}>
     {Array.from(Array(SLIDE_COUNT), (_, idx) => (
-      <Card height='300px' color='primary.light'>
-        <Flex border='primary' key={idx}>
-          Slide {idx}
+      <ToutCard height='300px' borderRadius={20} boxShadowSize='md' borderWidth={0}>
+        <BackgroundImage height='190px' width='100%' image='https://cdn2.thecatapi.com/images/dnn.jpg' />
+        <Flex color='background.lightest' p={[1, 1, 2, 2, 2, 3]}>
+          <Box>
+            <Text fontSize={[0, 0, 1, 1, 2]} fontWeight='bold' mb={2}>
+              City {idx}, AB
+            </Text>
+            <Text fontSize={[0, 0, 0, 0, 1]}>Hotel for 2 nights</Text>
+            <Text fontSize={[0, 0, 0, 0, 1]}>+ Round-trip flight</Text>
```

Comment:
should we use `[0, null, null, null, 1]`? It's more long winded but it creates less CSS media queries which could impact performance

Reply:
Updated